integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,14 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--memory-swappiness", "0", "busybox", "true")
```

Comment:
1. user specifies --memory-swappiness value => use value
2. user specifies --memory-swappiness =>  no value specified use deafult = 60
3. user does not specify --memory-swappiness => inherit the value from parent
I could not differentiate between (2) and (3) ..
@hqhq @lizf-os @cpuguy83 please let me know if you have any suggestions
Note that current behaviour is unless we override it uses whatever is inherited from parent..
so except documentation change rest of the code is in sync with what we expect.